### ISSUE TITLE
why the results are different from those in the paper

### ISSUE THREAD

**Issue by Vincent-Hoo** [ilovepose/fast-human-pose-estimation.pytorch#12]
In the paper, the hourglass trained with distillation achieved 90.9, while in this repo the result is 89. Can you explain the difference?

**Comment by xizero00** [comment]
There are three reasons:
(1) The PCKh metric implemented in the HRNet codebase for MPII dataset is slightly different from that in our paper, the PCKh metric in the HRNet codebase is not official.
(2) This repo re-implemented FPD using pytorch, the pytorch is different from the torch.
(3) We use the multiple-scale search testing in the original torch implementation of our paper.
However, these do not affect the evaluation of the effectiveness of the FPD.
Hope it will help you.

Official PCKh metric:
http://human-pose.mpi-inf.mpg.de/results/mpii_human_pose/evalMPII.zip

Multiple-scale search is used in PyraNet:
https://github.com/bearpaw/PyraNet/blob/master/train.lua#L224-L275
https://github.com/bearpaw/PyraNet/blob/master/dataloader.lua#L77-L139
